spec/lib/life_game/cell_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../../spec_helper'
+
+RSpec.describe Cell do
+  describe '#alive?' do
+    subject { cell.alive? }
+    context "when State is DEAD" do
+      let(:cell) { Cell.new(0, 0, false) }
+      it { is_expected.to be false }
+    end
+
+    context "when State is ALIVE" do
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.